apps/anoma_lib/lib/anoma/rm/intent.ex

```diff
@@ -22,4 +25,9 @@ defimpl Anoma.RM.Intent, for: Anoma.RM.DumbIntent do
   def verify(%DumbIntent{} = intent) do
     intent.value == 0
   end
+
+  @impl true
+  def nullifiers(%DumbIntent{}) do
+    MapSet.new([])
```

Comment:
Should we return a dummy nullifier/s here?

Reply:
An empty set is good enough. "DumbIntent" is absolutely fake, which was around for just initial testing. It does not have a concept of nullifiers really, so making it always return an empty set is good enough.

Reply:
Cool, thanks for your response!